raiden/raiden_service.py

```diff
@@ -687,7 +687,7 @@ def install_all_blockchain_filters(
             )
 
             for token_network in token_networks:
-                token_network_proxy = self.chain.token_network(token_network)
+                token_network_proxy = self.chain.token_network(typing.Address(token_network))
```

Comment:
I don't know what's the best way to proceed here.  The callee looks like:

```
class TokenNetwork:
    def __init__(
            self,
            jsonrpc_client,
            manager_address, # <---- So it's just an Address?
            contract_manager: ContractManager,
    ):
```

Reply:
Ah yes. This is a legacy name that got left over. `manager_address` refers to the old `ChannelManager` contract we had in the past. In this case you can go ahead and rename it to `token_network_address` as this is what this is. You can verify what I am saying by looking at what the constructor of `TokenNetwork` does with this address.

TL;DR: Change type to `TokenNetworkAddress`

Reply:
Done.  This PR is now 80 lines bigger.  That's a good thing.